### ISSUE TITLE
Feature Request: List of Current Active Users as we have on Web App

### ISSUE THREAD

**Issue by adambirds** [zulip/zulip-mobile#4575]
Would love for us to be able to see a list of current active users, as would be extremely useful to know if the person you want to talk to is online or not without having to have them on a previous direct message.

**Comment by abhi0504** [comment]
@adambirds yeah i guess there must be `presence status indicator` in PM conversation too I would like to work on this issue 
😊

**Comment by abhi0504** [comment]
@chrisbobbe @gnprice  whats your suggestions over this issue

**Comment by abhi0504** [comment]
@adambirds if we add status indicator for offline users it looks like this 
![Android Emulator - Pixel_3_API_29_5554 03-04-2021 15_29_57 (2)](https://user-images.githubusercontent.com/56453541/113475266-1f86dc00-9492-11eb-8526-143565ab19da.png)
Is it ok ?

**Comment by im-adithya** [comment]
@adambirds We already have this feature.
You can navigate to the Search bar in the PMs tab to find a list of all the Active, Idle, and Offline users!

**Comment by abhi0504** [comment]
Also @im-adithya I think we can add this grey box for the offline case
